consumer/pdata/common.go

```diff
@@ -36,13 +36,13 @@ func (ts TimestampUnixNano) String() string {
 type AttributeValueType int
 
 const (
-	AttributeValueNULL = iota
+	AttributeValueNULL AttributeValueType = iota
```

Comment:
Why not using "AttributeValueNull" as name? Maybe in a separate PR.

Reply:
Do you mean to avoid UPPERCASE?

Reply:
Yes

Reply:
It is an exported name used in contrib. I am not sure we want to touch it. Certainly not in this PR, which only adds arrays. Perhaps in a future PR if we want it.

Reply:
I would be happy to review a future PR that does this